### PR TITLE
added links to view About Us page, links to navigate back to the home…

### PR DESCRIPTION
**Pull request by JJSanch**
… page from header & footer, and added cards on About Us page

### CODE REVIEW

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ class App extends React.Component {
             let recipe = this.state.recipes.find(recipe => recipe.id === +id)
             return <RecipeShow recipe={ recipe } /> }} />
             <Route path="/userrecipes" component={ UserRecipes } />
-             <Route path="/aboutus" component={ AboutUs } />
+          <Route path="/aboutus" component={ AboutUs } />
```
Comment (minachuong) [inline app/javascript/components/App.js]: the indentation of this route should be the same as the route below

File: app/javascript/components/pages/AboutUs.js [hunk 2]
```diff
@@ -1,11 +1,113 @@
...
+//       return(
+//       <>
+//         <h1>About Us</h1>
+//       </>
+    
+         
+//       )    
+//   }
+// }
+
```
Comment (minachuong) [inline app/javascript/components/pages/AboutUs.js:113]: remove commented code if not used

**Comment by JJSanch** [comment]
https://trello.com/c/IPIh9DPv

**Review by minachuong — APPROVED** [review]
👍 keep files clean by removing commented code

File: app/javascript/components/App.js (hunk 1 above)
Comment (csullivan710) [reply app/javascript/components/App.js]: Done

File: app/javascript/components/pages/AboutUs.js (hunk 2 above)
Comment (csullivan710) [reply app/javascript/components/pages/AboutUs.js:113]: I believe this may be used in the future

Comment (JJSanch) [reply app/javascript/components/pages/AboutUs.js:113]: got it

**Pull request merged**